src/pyflex/window_selector.py

```diff
@@ -186,6 +189,18 @@ def default(self, obj):
             except TypeError:
                 filename.write(j.encode())
 
+    def write_text(self, filename):
+        """
+        Write plain text file for the window output
+        """
```

Comment:
Can you add a short example to the docstring? Also please document the `filename` parameters.